bin/src/cs.rs

```diff
@@ -57,6 +71,31 @@ fn convert_to_go_slices(vec: &Vec<String>) -> (GoSlice, Vec<GoString>) {
         go_slices,
     )
 }
+pub fn run_connect(connect_args: ConnectArgs) {
+    let mut args = if let Some(config) = connect_args.config {
+        vec!["connect".to_owned(), "-config".to_owned(), config]
+    } else {
+        vec!["connect".to_owned()]
+    };
+    let (args, go_str) = convert_to_go_slices(&args);
+    info!("Run connect cmd.");
+    let rt = tokio::runtime::Runtime::new().unwrap();
+    rt.block_on(async move {
+        let addr = "127.0.0.1:7000";
+        info!("Runtime started.");
+        let mut stream = tokio::net::TcpStream::connect(&addr).await.unwrap();
```

Comment:
@AthLw Could I check whether this stream only can connect 127.0.0.1 address? It looks like variable `args` isn't used in this function. And also is this command incomplete as TODO exists in the line 96?

/cc @vyloy

Reply:
change it from config file